### PR TITLE
[6]Designed a Product Page Using HTML

### PR DESCRIPTION
**Pull request by Iyiola-stack**
## Task: `Product page using HTML`
- `Created a new branch and a new product page
shot
shot2

`

### CODE REVIEW

File: submissions/peterIyiola/html/productPage-1.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+    <meta charset="UTF-8">
+    <title>Better Buys</title>
+    </head>
+<body>
+<nav>
+    <center><a href="#"><h1><b>Better Buys</b></h1></a>
```
Comment (ToluAkin) [inline submissions/peterIyiola/html/productPage-1.html:9]: The `center` tag is not supported in HTML5. You need to remove it.

File: submissions/peterIyiola/html/productPage-1.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+    </ul>
+    </div>
+
+    <section>
+    <!--Product Images and Description-->
+<div>
+    <ul>
+    <li>
+        <a href="#"><img src="TVs.jpg" alt="Smart TV" height="20%" width="20%"></a>
```
Comment (ToluAkin) [inline submissions/peterIyiola/html/productPage-1.html:40]: Remove the `height` and `width` on the image

File: submissions/peterIyiola/html/productPage-1.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        <p>65 inches QLED SMART TV 
+            <br>
+        +Wall Brackets and 12 Months warranty<br>
+        Price = 250,000</p>
+        
+        </li>
+
+    <li>
+        <a href="#"><img src="HT.jpg" alt="Home Theatre" height="20%" width="20%"></a></<a>
```
Comment (ToluAkin) [inline submissions/peterIyiola/html/productPage-1.html:49]: same as above

File: submissions/peterIyiola/html/productPage-1.html (hunk 1 above)
Comment (Iyiola-stack) [reply submissions/peterIyiola/html/productPage-1.html:9]: okay

File: submissions/peterIyiola/html/productPage-1.html [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+               <li>
+               LG LED 55 Inches ANdroid TVs
+               </li>
+               <li>Hisense HD Sound System</li>
+               <li>Polystar Double Door Refrigerator/li>
+               <li>Scanfrost Air Purifier Air conditioner</li>
+               </ol>
+               </P>
+               </h2>
+               </nav>
```
Comment (ToluAkin) [inline submissions/peterIyiola/html/productPage-1.html:80]: Why is your `nav` ending here. Format this file appropriately. Check the slides on how to format html files.

File: submissions/peterIyiola/html/productPage-1.html [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+                  <p>HisenseRegrigerators<br>
+                  Price =200,000 
+                  <br>
+                  Hisense 24Litres Refrigerator<br>
+                  Buttom Freezer and Water Dispenser
+                  </p>
+               </ul>
+               <nav>
+               <h4>
+               <P>
```
Comment (ToluAkin) [inline submissions/peterIyiola/html/productPage-1.html:68]: Paragraph tags are not in uppercase. Update the begin and end tag.

File: submissions/peterIyiola/html/productPage-1.html [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+                  Price=170,000</p>
+                  <hr>  
+                  <p>HisenseRegrigerators<br>
+                  Price =200,000 
+                  <br>
+                  Hisense 24Litres Refrigerator<br>
+                  Buttom Freezer and Water Dispenser
+                  </p>
+               </ul>
+               <nav>
```
Comment (ToluAkin) [inline submissions/peterIyiola/html/productPage-1.html:66]: Why do you have more than one `nav` tag. You're are suppose to have only one in an html file.

**Comment by Iyiola-stack** [comment]
At last!!

**Pull request merged**